src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java

```diff
@@ -53,8 +51,6 @@ public void testUnLoggedCommandLineSexChrs() throws IOException {
         final String sampleName = "HCC1143";
         File tDir = IOUtils.tempDir("Test", "Plotting");
         System.out.println(tDir);
```

Comment:
also note that i removed these in #330, so if you decide to log, then rebase accordingly.  although it looks to me like this was just debugging stuff that got left behind?